components/local-app/cmd/workspace-ssh.go

```diff
@@ -15,7 +15,17 @@ var dryRun bool
 var workspaceSSHCmd = &cobra.Command{
 	Use:   "ssh <workspace-id>",
 	Short: "Connects to a workspace via SSH",
-	Args:  cobra.ExactArgs(1),
+	Args:  cobra.MinimumNArgs(1),
+	Example: `  # connect to workspace with current terminal session
+  $ gitpod workspace ssh <workspace-id>
+
+  # Execute with ssh command
+  $ gitpod workspace ssh <workspace-id> -- ls -la
+  $ gitpod ws ssh <workspace-id> -- -t watch date
+
+  # Get all SSH features with --dry-run
```

Comment:
Why do we need it? Since we pass all ssh args everything should be supported already?

Reply:
That unsupported case mentioned in PR description should work with dry run

Reply:
Is not it matter of doings something like `command.Stdin = os.Stdin` in code?

Reply:
I see we already do it